packages/rest/src/coercion/validator.ts

```diff
@@ -65,8 +65,25 @@ export class Validator {
   isAbsent(value: any) {
     if (value === '' || value === undefined) return true;
 
+    const spec: ParameterObject = this.ctx.parameterSpec;
     const schema: SchemaObject = this.ctx.parameterSpec.schema ?? {};
-    if (schema.type === 'object' && value === 'null') return true;
+    const valueIsNull = value === 'null' || value === null;
+
+    // if parameter spec contains schema object, check if supplied value is NULL
+    if (spec.schema) {
+      if (schema.type === 'object' && valueIsNull) return true;
+    } else {
+      // for url encoded Json object query parameters, schema is defined under content['application/json']
+      if (
+        spec.in === 'query' &&
+        spec.content &&
+        spec.content['application/json'] &&
+        spec.content['application/json'].schema
```

Comment:
Is it possible to use https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining?

Can we have a helper method to check this case?

Reply:
+1 for Raymond's comment, try
```ts
if (spec.content?.['application/json']?.schema) {
//
}
```
?

Reply:
@raymondfeng @jannyHou I have implemented as per your suggestions with optional chaining. please verify.